docs/contents/releases/1-21/1/index.md

```diff
@@ -0,0 +1,6 @@
+# EKS-D v1-21-eks-1 Release
+
+For additional information, see the [changelog](CHANGELOG-v1-21-eks-1.md) for this release.
+
+## Release Manifest
+Download the release manifest here: [kubernetes-1-21-eks-1.yaml](https://distro.eks.amazonaws.com/kubernetes-1-21/kubernetes-1-21-eks-1.yaml)
```

Comment:
We've changed how we format this doc a little. I'll explain offline what we'll need to do to update it.